src/components/EventCard.js

```diff
@@ -13,6 +15,21 @@ const EventCard = ({ row }) => {
   const featureImage = row.values.feature_image;
   const eventImage = featureImage ? featureImage.url : '';
 
+  function checkAttendanceType(type) {
+    return row.values.attendance_type.some(o => o.name === type);
+  }
+
+  function renderAttendanceType() {
+    let label = [];
+    if (checkAttendanceType('virtual')) {
+      label.push('Virtual');
+    }
+    if (checkAttendanceType('in-person')) {
+      label.push('In-Person');
+    }
```

Comment:
Did you intend for this PR to have changes from #70 ?

Reply:
@gcorne: I did intend this PR to have changes from #70. I was trying to avoid merge conflicts, once I merge in #70. I'm happy to change this, if you'd like.